guides/common/modules/con_shellhooks-plugin.adoc

```diff
@@ -1,25 +1,34 @@
 [id="shellhooks_{context}"]
 = Shellhooks
 
-With webhooks, one {Project} event can only be mapped to one API call.
+With webhooks, you can only map one {Project} event to one API call.
```

Comment:
I am confused. The title is "Shellhooks", but you're immeditely starting with "webhooks".

Reply:
Yes, this is to show the difference between webhooks and shellhooks. Shellhooks improve on webhooks because they can map multiple events to one API call as stated further below.